src/SFA.DAS.AssessorService.Application.Api/Validators/EpaOrganisationValidator.cs

```diff
@@ -687,6 +686,21 @@ public ValidationResponse ValidatorWithdrawOrganisationRequest(WithdrawOrganisat
             return validationResult;
         }
 
+        public ValidationResponse ValidatorWithdrawStandardRequest(WithdrawStandardRequest request)
```

Comment:
Typo
```suggestion
        public ValidationResponse ValidateWithdrawStandardRequest(WithdrawStandardRequest request)
```

Reply:
All the other methods use "Validator" - will leave as it is for consistency...